.github/workflows/pki-ca-test.yml

```diff
@@ -527,6 +527,165 @@ jobs:
 
           diff expected stderr
 
+      - name: Revoke sslserver certificate
+        run: |
+          SERIAL=$(docker exec pki  pki -d /etc/pki/pki-tomcat/alias nss-cert-show sslserver | sed -ne 's/^  Serial Number: \(.*\)$/\1/p')
+          docker exec -i pki pki -n caadmin ca-cert-hold --force  $SERIAL
+
+      - name: Check client1 with already trusted server cert revoked
+        run: |
+          # run PKI CLI
+          docker exec client1 pki \
+              -U https://pki.example.com:8443 \
+              info \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # check stdout
+          cat > expected << EOF
+            Server URL: https://pki.example.com:8443
+            Server Name: Dogtag Certificate System
+            Server Version: 11.7.0
+          EOF
+
+          diff expected stdout
+
+          # check stderr
+          diff /dev/null stderr
+
+      - name: Check client2 with already trusted server cert revoked
+        run: |
+          # run PKI CLI
+          docker exec client2 pki \
+              -D org.dogtagpki.client.socketFactory=org.dogtagpki.client.NonBlockingSocketFactory \
+              -U https://pki.example.com:8443 \
+              info \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # check stdout
+          cat > expected << EOF
+            Server URL: https://pki.example.com:8443
+            Server Name: Dogtag Certificate System
+            Server Version: 11.7.0
+          EOF
+
+          diff expected stdout
+
+          # check stderr
+          diff /dev/null stderr
+
+      - name: Check client1 with trusted root and revoked server cert
+        run: |
+          # remove nss DB
+          docker exec client1 rm -rf /root/.dogtag
+
+          # init nss DB with trusted root
+          docker exec client1 pki nss-cert-import \
+              --cert $SHARED/ca_signing.crt \
+              --trust CT,C,C \
+              ca_signing
+
+          docker exec client1 pki nss-cert-find
+
+          # run PKI CLI
+          docker exec client1 pki \
+              -U https://pki.example.com:8443 \
+              info \
+              > >(tee stdout) 2> >(tee stderr >&2) || true
+
+          # check stdout
+          cat > expected << EOF
+            Server URL: https://pki.example.com:8443
+            Server Name: Dogtag Certificate System
+            Server Version: 11.7.0
+          EOF
```

Comment:
I suppose it should have failed. If this is a known bug could you add a link to the issue? Or if it requires further investigation please add a TODO.

Reply:
Not really. [Default OCSP policy in CryptoManager](https://github.com/dogtagpki/jss/blob/31699ded207d4c3798aaeb4a05ba9a0cf236b240/base/src/main/java/org/mozilla/jss/CryptoManager.java#L1399) is to not check certificate verification. In tomcatJSS there is the `revocationCheck` (formerly `enableOCSP`) to enable the check. I do not see it enabled  in the DefaultSocketFactory  so it works as expected.

Reply:
Hmm.. but this is on the client side. Maybe we should always enable revocation check on the client side so the behavior is consistent?

Reply:
We are migrating to JSSSocket and the JSSTrustManager where OCSP check can be enabled independently of CryptoManager configuration. PKI client code can be modified to enable OCSP check but is there a reason to modify the current implementation? My understanding was to replace SSLSocket.